source/stylesheets/refills/unstyled/_tooltip.scss

```diff
@@ -0,0 +1,36 @@
+.tooltip-item {  
+  display: inline-block;
+  position: relative;
+  text-align: center;
+
+  &:focus,
+  &:hover .tooltip {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .tooltip {
+    @include position(absolute, null 0 3em 0);
+    @include transition (all 0.2s ease-in-out);
+    background: whitesmoke;
+    width: 12em;
```

Comment:
Properties should be ordered background, margin-left, opacity, text-align, visibility, width, z-index